docs/wdc_authentication.md

```diff
@@ -73,7 +73,7 @@ The auth phase will be displayed by Tableau in two scenarios:
     from an API can expire or be revoked.  In that scenario, before fetching data, the WDC would want to call this method
     in order to re-authenticate the user.  For more information, see the **OAuthProxyExample** connector included in the SDK. 
     
-    **IMPORTANT:** This function must be called from the init method during the gather data phase.  Additionally, this function only works when called from a fresh instance of Tableau.  I.e. if you call this immediately after creating the connection, the call will fail.  This is not ideal behavior, but the intended scenario for this function is when called from opening a workbook from scratch (when the auth token is no longer available).  
+    **IMPORTANT:** The `abortforauth` function must be called from the `init` method during the gather data phase. Additionally, this function only works when called from a fresh instance of Tableau. That is, if you call this immediately after creating the connection, the call will fail.  This is not ideal behavior, but the intended scenario for this function is when called from opening a workbook from scratch (when the auth token is no longer available).  
```

Comment:
use `abortForAuth`

Reply:
also, I'm not sure what " a fresh instance of Tableau" means here

Reply:
Yes.  The "fresh instance"  is confusing.   From the scenario, it sounds like the `abortForAuth` function only works under very specific conditions.  I am also not clear on what those conditions are.  Sounds like it only works when you start up Tableau and open an existing workbook that has a WDC.  The OAuth example code has this snippet in the `init()` function. 

``` 
        if (tableau.phase == tableau.phaseEnum.gatherDataPhase) {
        // If API that WDC is using has an endpoint that checks
        // the validity of an access token, that could be used here.
        // Then the WDC can call tableau.abortForAuth if that access token
        // is invalid.
      }
```

Reply:
I'm shortening it to this: 

 **IMPORTANT:** The `abortForAuth` function must be called from the `init` method during the gather data phase. One intended scenario for this function is when a user, who has not been authenticated, opens an existing workbook and attempts to refresh data. In this case, the auth token is no longer available.

Reply:
sounds reasonable to me. thanks.